src/reanimated2/Hooks.js

```diff
@@ -224,6 +237,7 @@ function styleUpdater(
   'worklet';
   const animations = state.animations || {};
   const newValues = updater() || {};
+  validateAnimatedStyles(newValues);
```

Comment:
I think it's enough to check it on RN side during the initial run. Let's not slow down UAS.

Reply:
That was my first idea. Then I realized someone could conditionally return an invalid value checking for `_WORKLET`. So I just handled this. I know, however, this is a rare edge case. I'm ok with checking it only on the initial run.

Reply:
fixed f3d2a5e